example/albums/migrations/0002_auto_20210720_2132.py

```diff
@@ -0,0 +1,25 @@
+# Generated by Django 3.2.5 on 2021-07-20 21:32
```

Comment:
Is this migration really needed ? I don't see any changes to the models of example app but maybe I'm missing something...

Reply:
It adds the 'ordering' (no idea why it didn't appear in the initial migration).  I can remove it if preferred - probably doesn't make any difference in the test app.

1st migration:

```
            options={
                'verbose_name': 'Album',
                'verbose_name_plural': 'Albums',
            },
```

new migration:

```
        migrations.AlterModelOptions(
            name='album',
            options={'ordering': ['name'], 'verbose_name': 'Album', 'verbose_name_plural': 'Albums'},
        ),
```

Reply:
OK thanks, we'll keep it